modules/rostests/kmtests/ntos_io/IoDeviceInterface.c

```diff
@@ -1,125 +1,190 @@
 /*
- * PROJECT:         ReactOS kernel-mode tests
- * LICENSE:         LGPLv2+ - See COPYING.LIB in the top level directory
- * PURPOSE:         Kernel-Mode Test Suite Device Interface functions test
- * PROGRAMMER:      Filip Navara <xnavara@volny.cz>
+ * PROJECT:     ReactOS kernel-mode tests
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Test for Device Interface functions
+ * COPYRIGHT:   Copyright 2011 Filip Navara <xnavara@volny.cz>
+ *              Copyright 2021 Mark Jansen <mark.jansen@reactos.org>
```

Comment:
When I do a search in the Git Log with this file for Mark Jansen, I am not able to find any entries. But, Git Blame shows almost all of the recent commits from Thomas Faber. Please review and correct. Thanks.

Reply:
Done. Added Thomas in copyrights as well.